persistence-jdbc/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/jdbc/JdbcPersistenceComponents.scala

```diff
@@ -30,7 +30,7 @@ trait JdbcPersistenceComponents
     with ReadSideJdbcPersistenceComponents
     with WriteSideJdbcPersistenceComponents
 
-private[lagom] trait SlickProviderComponents extends DBComponents {
```

Comment:
Why did you remove `private[lagom]`?

Reply:
Users that want to use Lagom with Akka Typed and Akka Projection can use this trait directly instead of `ReadSideJdbcPersistenceComponents` or `WriteSideJdbcPersistenceComponents` which bring a things they don't need.